cpp/src/IceBox/ServiceManagerI.cpp

```diff
@@ -63,6 +63,17 @@ namespace
         string entryPoint;
         Ice::StringSeq args;
     };
+
+    inline bool printStackTraces(const PropertiesPtr& properties)
+    {
+#ifdef NDEBUG
+        // Release build
+        return properties->getIcePropertyAsInt("Ice.PrintStackTraces") > 0;
+#else
+        // Debug build
+        return properties->getPropertyAsIntWithDefault("Ice.PrintStackTraces", 1);
```

Comment:
```suggestion
        return properties->getPropertyAsIntWithDefault("Ice.PrintStackTraces", 1) > 0;
```

Reply:
Fixed as suggested (also in Instance.cpp)